extensions/divina.md

```diff
@@ -172,7 +179,158 @@ This current draft does not cover guided navigation over alternate versions of e
+
+### transitionForward, transitionBackward
+
+Keep in mind that a forward transition is placed on the TARGET resource of the transition.
+
+### The Transition Object
+
+This specification defines the following keys for this JSON object:
+
+| Key  | Definition | Format | Required? |
+| ---- | -----------| -------| ----------|
+| [type](#type)  | Type of transition  | `cut`, `dissolve`, `slide-in`, `slide-out`, `push`, `animation` | Yes |
+| [direction](#direction)  | Direction of a slide-in, slide-out or push transition  | `ltr`, `rtl`, `ttb`, `btt` | Yes if the type is slide-in, slide-out or push. |
+| [sequence](#sequence)  | Sequence of images which create an animation | Array of URIs | No |
+| [file](#file)  | Video file which creates an animation | URI | No |
+| [duration](#duration)  | The duration of the transition in milliseconds | Integer | No |
+
+
+#### type
+
+| Value  | Definition | 
+| ---- | -----------| 
+| `cut` | the new resource immediately replaces the current one; this transition does not need a duration and is only useful when continuous=true, as a way to specify an explicit cut (“page change”) between two resources. | 
+| `dissolve` |  the new resource appears above the current one, with opacity increasing from 0 to 1. | 
+| `slide-in` | the new resource appears above the current one, and moves into the viewport to cover it. | 
+| `slide-out` | the next resource is placed below the current one, which moves out of the viewport. | 
+| `push` |  the next resource moves into the viewport while the current one moves out of it, revealing the next one below. | 
+| `animation` | the next resource appears after playing a sequence of images or a video. |
+
+If the transition type is `animation`, either a `file` or a `sequence` property <strong class="rfc">must</strong> be specified.
+
+#### direction
+
+| Value  | Definition | 
+| ---- | -----------| 
+| `ltr` | the new image comes from the left (whether the transition is forward or backward). | 
+| `rtl` | the new image comes from the right. | 
+| `ttb` | the new image comes from the top . | 
+| `ltr` | the new image comes from the bottom. | 
+
+Note: Usually, if the reading progression is ltr, forward transitions will be rtl. Also, the reverse of an rtl slide-in is an ltr slide-out.
+
+#### sequence
+
+Only used when the type is `animation`, the value of the `sequence` property is an array of Link Objects pointing to bitmap images displayed before the next resource appears.
+
+Each image in the array is a frame displayed for a slice of `duration` divided by the number of images of the sequence.
+
+In this case `overflow` is forced to `clipped` while `fit` is inherited from the parent resource (i.e. the target resource for a forward transition and the source for a backward one).
```

Comment:
This is problematic as well.

Reply:
Good catch. I have suppressed the 2 sentences from the spec, reworded the recommendation and moved it to the guidelines.